compiler/src/error.rs

```diff
@@ -40,6 +40,19 @@ pub enum CompileErrorType {
     InvalidYield,
 }
 
+impl CompileError {
+    pub fn is_tab_error(&self) -> bool {
+        if let CompileErrorType::Parse(parse) = &self.error {
+            if let ParseErrorType::Lexical(lex) = parse {
+                if let LexicalErrorType::TabError = lex {
```

Comment:
You could do

```rust
if let CompileErrorType::Parse(ParseErrorType::Lexical(LexicalErrorType::TabError)) = self.error { true } else { false }
```

Reply:
I could, but I don't see a need to change my code.